tests/AnalyzeSpec.hs

```diff
@@ -1017,6 +1020,59 @@ spec = describe "analyze" $ do
           |]
     expectPass code $ Valid Success'
 
+
+  describe "create-capability-guard: success" $ do
+    let code =
+          [text|
+            (defcap CAP (i:integer)
+              true)
+
+            (defun test:bool ()
+              (with-capability (CAP 1)
+                (enforce-guard (create-capability-guard (CAP 1)))))
+          |]
+    expectPass code $ Valid Success'
+
+  describe "create-capability-guard: failure" $ do
+    let code =
+          [text|
+            (defcap CAP (i:integer)
+              true)
+
+            (defun test:bool ()
+              (with-capability (CAP 2)
+                (enforce-guard (create-capability-guard (CAP 1)))))
+          |]
+    expectPass code $ Valid Abort'
+
+  -- sadly doesn't look like pact guard func can really do much in FV.
+  xdescribeWith "TODO fix pact guard FV" "create-capability-pact-guard: failure" $ do
+    expectFalsified [text|
+        (defcap CAP (i:integer)
+          true)
+
+        (defpact testpact ()
+          (step
+            (with-capability (CAP 2)
+              (enforce-guard
+                (create-capability-pact-guard (CAP 1)))))
+          (step true))
+        |]
+
+  -- same issue here. TODO make pact guards work in FV.
```

Comment:
Future, the current state of pact-guard support in FV is either broken or intentionally nonfunctional. FV doesn't "run guards" so it's not clear what should be done, and defpact support overall is shaky.

Reply:
A TODO in code like this is fairly equivalent to not writing it, so I'd rather see a link to a GitHub issue if it is to be scheduled for work in the future; otherwise, let's make a catch-all for "known FV issues" and add it to a textual list there.

Reply:
#1061 created and referenced in code.

Reply:
Thank you!